Documentation/gettingstarted/local-redirect-policy.rst

```diff
@@ -301,5 +309,233 @@ configuring the CiliumLocalRedirectPolicy.
+cluster, and intercepts requests going to the AWS metadata server to fetch
+security credentials for pods.
+
+- In order to only redirect traffic from pods to the kiam agent, and pass
+  traffic from the kiam agent to the AWS metadata server without any redirection,
+  we need the socket lookup functionality in the datapath. This functionality
+  requires v5.1.16, v5.2.0 or more recent Linux kernel. Make sure the kernel
+  version installed on EKS cluster nodes satisfies these requirements.
+
+- Deploy kiam.
+
+  .. tabs::
+
+    .. group-tab:: Quick Start
+
+            Deploy kiam using the example yaml file.
+
+            .. parsed-literal::
+
+                $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/kiam.yaml -n kube-system
+
+    .. group-tab:: Manual Configuration
+
+        - Deploy `kiam <https://github.com/uswitch/kiam>`_ using helm charts.
+            .. code-block:: bash
+
+                $ helm repo add uswitch https://uswitch.github.io/kiam-helm-charts/charts/
+                $ helm repo update
+                $ helm template kiam uswitch/kiam > kiam.yaml
+
+        - Edit the kiam yaml file, and remove the "--iptables" argument passed to the ``kiam-agent Daemonset``.
+
+        - Make sure the ``kiam-agent Daemonset`` is run in the ``hostNetwork`` mode.
+
+        - Apply the kiam configuration.
+
+            .. code-block:: bash
+
+                $ kubectl apply -f kiam-helm.yaml -n kube-system
+
+        .. note::
+
+            - The ``addressMatcher`` ip address in the local redirect policy is set to
+              the ip address of the AWS metadata server and the ``toPorts`` port
+              to the default HTTP server port. The ``toPorts`` field under
+              ``redirectBackend`` configuration in the policy is set to the port that
+              the kiam agent listens on. The port is passed as "--port" argument in
+              the ``kiam-agent Daemonset``.
+            - The local redirect policy namespace is set to the namespace
+              in which kiam-agent Daemonset is deployed.
+
+
+- Deploy the local redirect policy to redirect pod traffic to the deployed kiam agent.
+
+    .. parsed-literal::
+
+        $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes-local-redirect/kiam-lrp.yaml
+
+
+
+- Once all the kiam agent pods are in ``Running`` state, the metadata requests
+  from application pods should get redirected to the node-local kiam agent pods.
+  You can verify this by running ``cilium monitor`` with ``related-to`` set to
+  the endpoint id of one of the application pods.
+
+Miscellaneous
+=============
+When a local redirect policy is applied, cilium BPF datapath translates frontend
+(ip/port/protocol tuple) from the policy to a node-local backend pod selected
+by the policy. However, such translation is skipped for traffic that originates
+from the backend and is destined to the frontend.
+
+
+
```

Comment:
The KIAM section seems to be duplicated at the moment.

Minor extra nit, this currently has 5 empty lines at the end of the file which are unnecessary and can be removed.

Reply:
>The KIAM section seems to be duplicated at the moment.

Looks like a result of bad merge? Fixed.

Reply:
@aditighag did you push the fixed version?

Reply:
Not yet, fixing the bad merge.

Reply:
Yes, pushed the fixed version. PTAL.